src/main/java/uk/gov/hmcts/reform/civil/enums/ClaimantResponseScenarioFlag.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.hmcts.reform.civil.enums;
+
+public enum ClaimantResponseScenarioFlag {
```

Comment:
Why can we not use the MultiPartyScenario enum here instead of repeating similar concept in this new enum

Reply:
Done